awscodecommit/main.go

```diff
@@ -259,17 +265,28 @@ func (clients Clients) preparePullRequests() ([]*codecommit.PullRequest, error)
 
 //sendPush sends an event containing data about a git commit that was pushed to a branch
 func (clients Clients) sendCommitEvent(commit *codecommit.Commit) error {
+
 	log.Info("send Commit Event")
 
-	codecommitEvent := PushMessageEvent{
-		Commit:           commit,
-		CommitRepository: aws.String(repoNameEnv),
-		CommitBranch:     aws.String(repoBranchEnv),
-		EventSource:      aws.String("aws:codecommit"),
-		AwsRegion:        aws.String(awsRegionEnv),
+	event := cloudevents.Event{
+		Context: cloudevents.EventContextV03{
+			Type:            "AWS CodeCommit commit event",
```

Comment:
any reason you're not using a reverse DNS name as the spec suggests?

Reply:
as in: https://github.com/cloudevents/spec/blob/v0.3/spec.md#type ? com.amazon.sqs

we are also missing `specVersion` I guess

Reply:
@sebgoa `specVersion` is automatically added by this function `AsV03()` which is why I did not include it in the event description. 

Regarding reverse DNS name, will add it as well

Reply:
do you have an example one of these events from AWS? I tried to find one via google but couldn't. That will help me with a recommendation for what the reverse DNS name should be.

Reply:
@duglin , maybe adding actual event type would be OK? like `commit` or `merge` etc?

Reply:
yep - exactly! using `commit` or `merge` at the end of the reverse DNS name is exactly what CE is expecting there.